src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -12,38 +12,23 @@
 namespace Symfony\Component\Form\Extension\Core\Type;
 
 use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
-use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
 use Symfony\Component\Form\ChoiceList\Loader\IntlCallbackChoiceLoader;
 use Symfony\Component\Intl\Countries;
 use Symfony\Component\OptionsResolver\Options;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class CountryType extends AbstractType implements ChoiceLoaderInterface
+class CountryType extends AbstractType
 {
-    /**
-     * Country loaded choice list.
-     *
-     * The choices are lazy loaded and generated from the Intl component.
-     *
-     * {@link \Symfony\Component\Intl\Intl::getRegionBundle()}.
-     *
-     * @var ArrayChoiceList
-     *
-     * @deprecated since Symfony 4.1
-     */
-    private $choiceList;
-
     /**
      * {@inheritdoc}
      */
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'choice_loader' => function (Options $options) {
+            'choice_loader' => static function (Options $options) {
```

Comment:
these are for 4.4

Reply:
Remove from this PR, later I will do the changes for 4.4